lib/util/mime_utils.js

```diff
@@ -44,6 +44,9 @@ shaka.util.MimeUtils = class {
         components.push(mimeKey + '="' + value + '"');
       }
     });
+    if (stream.hdr == 'PQ') {
```

Comment:
The HLS specification allows: SDR, PQ, HLG. Should we also add HLG to the list?

Reply:
@michellezhuogg do you have any preference about it?

Reply:
I am not familiar with HLG and its according mimetype. Shall we skip HLG until we are sure about how to represent it with mimetype?

Reply:
OK, I think it's best to skip it for now. So this is ready to be merged whenever you want.